### DISCUSSION TITLE
Native Port Disconnected Issue

### DISCUSSION THREAD

**Discussion by J97Bqdfweasrf** [aclap-dev/video-downloadhelper#195] [Help and Q&A]
### Your question

I am trying to download a video from OnlyFans, but keep getting a Native Port Disconnected error. Is OnlyFans not a supported website???

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\B_Jon",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by paulrouget** [comment]
OnlyFan has started introducing DRM. That might be your issue. If you have a specific link that's failing, I can look into it.

**Reply by J97Bqdfweasrf** [reply]
That makes sense. Is there a way to bypass it? Is Fansly a supported site? The link is: https://onlyfans.com/945466622/khaosleon

**Reply by GrampaWildWilly** [reply]
![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/cff7004d-aaed-41a4-aa4c-b22abb040ad7)

There's your answer.  You can do this yourself.  Go into the browser settings & find where it enables or disables DRM support.  All the browsers should have such a setting.  Firefox is all I use & it has such a setting.  Just disable that setting, then reload the web page you're curious about.  If that message shows up, you're out of luck.  VDH has never, does not now, & will never download content protected by DRM.  Go find a screen recorder if you absolutely must have something protected by DRM.

Sites are not supported.  Types of media are supported.  So it's not a question of supporting Fansly.  Just try to get something & as long as it's not behind DRM protection, there is a chance -- A CHANCE -- VDH can get it.  But if you post something that requires a paid membership, nobody here is going to investigate it.  The VDH guys have repeatedly posted that (on our old Google Groups forum).  I, not an employee of VDH, take my cue from them.  I won't look into it.  And don't even offer to share your login credentials.  We (VDH & me) will not accept them.  Post screenshots to allow us virtually behind your login.  But keep your login credentials private, secret, not for public consumption.  I'm a member of neither OnlyFans nor Fansly, & I'm not about to join.  So I won't be of any help unless you report problems by posting screenshots.

**Reply by J97Bqdfweasrf** [reply]
Thank you for your time. I apologize for wasting it.

**Reply by GrampaWildWilly** [reply]
Now I'm going to get mad at you. 😃    Apologies aren't in order.  I did not waste my time.  Besides, any answer is here for anybody to read.  In the event somebody searches this forum & comes across this discussion, you never know who else might be helped by this discussion.  So no time was wasted.  Now if we can just get people to do searches before they post . . .
